crates/workspace/src/workspace.rs

```diff
@@ -941,6 +941,25 @@ impl Workspace {
                     );
                 }
 
+                project::Event::LanguageServerShowDocument(request) => {
+                    if request.external {
+                        cx.open_url(request.uri.as_str());
+                        let request = request.clone();
+                        cx.background_executor()
+                            .spawn(async move {
+                                request.respond(Ok(())).await;
+                            })
+                            .detach();
+                    } else {
+                        let request = request.clone();
+                        cx.background_executor()
+                            .spawn(async move {
+                                request.respond(Err(anyhow!("not implemented"))).await;
+                            })
```

Comment:
The LSP specification for the show document request response doesn't have a way to report why the request failed, so maybe we just log this error here as well?